### ISSUE TITLE
Serious accessiblity issues

### ISSUE THREAD

**Issue by SoniEx2** [MovingBlocks/legacy-website#117]
The website doesn't work well at all with text browsers and screen readers. For example, none of the IRC/discord/etc links have text descriptions, and they don't show up at all on text browsers.
